components/popover_list_members/popover_list_members_item/popover_list_members_item.tsx

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Client4} from 'mattermost-redux/client';
+import {UserProfile} from 'mattermost-redux/types/users';
+import {isGuest} from 'mattermost-redux/utils/user_utils';
+
+import ProfilePicture from 'components/profile_picture';
+import MessageIcon from 'components/widgets/icons/message_icon';
+import BotBadge from 'components/widgets/badges/bot_badge';
+import GuestBadge from 'components/widgets/badges/guest_badge';
+import SharedUserIndicator from 'components/shared_user_indicator';
+import CustomStatusEmoji from 'components/custom_status/custom_status_emoji';
+
+import {UserStatuses} from 'utils/constants';
+
+export type Props = {
+    showMessageIcon: boolean;
+    onItemClick: (user: UserProfile) => void;
+    status: string;
+    user: UserProfile;
+    displayName?: string;
+}
+
+PopoverListMembersItem.defaultProps = {
+    status: UserStatuses.OFFLINE,
+};
+
+export default function PopoverListMembersItem(props: Props) {
+    const handleClick = () => {
+        props.onItemClick(props.user);
+    };
+
+    if (!props.user) {
+        return null;
+    }
+
+    if (!props.displayName) {
+        return null;
+    }
```

Comment:
this can be combined:

```suggestion
    if (!(props.user || props.displayName)) {
        return null;
    }
```